src/Symfony/Component/Console/Tests/Helper/ProgressBarTest.php

```diff
@@ -763,6 +825,7 @@ public function testAnsiColorsAndEmojis()
         putenv('COLUMNS=156');
 
         $bar = new ProgressBar($output = $this->getOutputStream(), 15);
+        $bar->setRedrawFrequency(1);
```

Comment:
I've changed this here because there is racing condition in this test when going with usleep approach. Sometimes, last output is `< 3 sec` instead of `< 1 sec`.